docs/source/conf.py

```diff
@@ -43,7 +43,6 @@
 
 # Configure intersphinx magic
 intersphinx_mapping = {
-    'pydoctor': ('https://pydoctor.readthedocs.io/api/', None),
```

Comment:
this will be added later once we publish the API docs on Read The docs....I have a separate PR.

The Proof of Concept is already done